### ISSUE TITLE
Metadata summary in dataset download

### ISSUE THREAD

**Issue by DimEvil** [gbif/portal16#564]
Until today it is still not easy to easily read and digest all the metadata which was originally given when a dataset was published...

When a (aggregated) dataset is downloaded, it would be nice if some kind of summarized metadata document could be generated, derived from the original metadata provided when the data was published, and placed in the DwC-A.zip as a .txt document. 

It could be a text document listing

**Name dataset  | description | keywords | samplingdescription | license** 

Or maybe just listing the dataset urls (or doi), so the metadata is really easy to find for the user:

**Name dataset | gbifUrl or doi** 

This document should/could give the user a quick overview of the metadata of the data which was downloaded.  Actually this data is already given in the new portal download page, so maybe just include this information in the download.

**Comment by MortenHofft** [comment]
Thanks. So to repeat you suggest including a list of the individual datasets that contributed data as part of the zipped download?
Makes sense. I believe this issue is related https://github.com/gbif/portal16/issues/277

**Comment by DimEvil** [comment]
Yes! That would already be very helpful! Or include a text document with a summary of all the datasets metadata in the zip file. (I don't know what the best option is?  What would the end user prefer?)
